core/electron/src/backend/ElectronHost.ts

```diff
@@ -22,7 +22,10 @@ import { DialogModuleMethod, electronIpcStrings } from "../common/ElectronIpcInt
 // cSpell:ignore signin devserver webcontents copyfile unmaximize eopt
 
 class ElectronIpc implements IpcSocketBackend {
-  public addListener(channel: string, listener: IpcListener): RemoveFunction {
+  public addListener(
+    channel: string,
+    listener: (evt: any, ...args: any[]) => void, // There is mismatch between IPC Event and Electron.IpcMainEvent. Defining `evt` as any as a temporary fix

```

Comment:
Issue: https://github.com/iTwin/itwinjs-core/issues/7575

Reply:
`ElectronIpc` is not exported out of file, so `ElectronIpc.addListener` is not available anywhere else. Also, no changes to _core-electron.api.md_.